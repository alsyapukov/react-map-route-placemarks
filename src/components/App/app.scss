@import './reset.scss';
@import url('https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i&display=swap&subset=cyrillic,cyrillic-ext,latin-ext');

body {
  font-family: 'Roboto', sans-serif;
  font-size: 16px;
}

.app {
  width: 100%;
  height: 100vh;
  background-color: #F9F9F9;
  display: flex;
  // justify-content: center;

  &__wrap {
    width: 100%;
    height: 100%;
    // margin-top: 200px;
    // margin-bottom: 200px;
  }
}