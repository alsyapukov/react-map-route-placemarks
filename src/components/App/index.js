import React, { Component } from "react";
import "./app.scss";

import EditRoute from '../EditRoute'

class App extends Component {
  render() {
    return (
      <div className="app">
        <div className="app__wrap">
          <EditRoute />
        </div>
      </div>
    );
  }
}
export default App;