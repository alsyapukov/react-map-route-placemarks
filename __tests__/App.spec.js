import React from 'react';
import App from '../src/components/App'
import { create } from 'react-test-renderer';

describe('Test App component', () =>{

  it('render App component', () => {
    const tree = create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });

})