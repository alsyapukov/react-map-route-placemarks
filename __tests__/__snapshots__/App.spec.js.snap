// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App component render App component 1`] = `
<div
  className="app"
>
  <div
    className="app__wrap"
  >
    <div
      className="edit-route"
    >
      <div
        className="edit-route__options"
      >
        <div
          className="search"
        >
          <div
            className="search__textbox"
          >
            <div
              className="textbox"
            >
              <input
                className="textbox__inner"
                onChange={[Function]}
                onKeyPress={[Function]}
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            className="search__list"
          >
            <ul />
          </div>
        </div>
      </div>
      <div
        className="edit-route__map"
      >
        <div
          className="yandex-map"
        >
          <div
            id="yandexMap"
          >
            <div
              className="yandex-map__loading"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
